modules/ROOT/pages/afm-upload-monitor.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can upload a suite of tests that you have written and want to run as a monitor, viewing the results the information displays in the *Functional Monitoring* section of Anypoint Monitoring.
+You can upload a suite of tests that you have written and want to run as a monitor, viewing the results the information displays in the *Functional Monitoring* section of Anypoint Monitoring. When you create monitors manually, you can create more complex test cases by using features such as the DataWeave expression language and context variables for saving values between calls to endpoints. 
```

Comment:
This is a bit unwieldy, given the three user actions already in the beginning of the sentence: "viewing the results the information displays in the *Functional Monitoring* section of Anypoint Monitoring." I crave minimalism; it's my simple mind!
```suggestion
To create your own monitor, uploading a suite of tests that you have written to obtain more specific or complex results. You can then run the tests and view the results in the *Functional Monitoring* section of Anypoint Monitoring. By creating monitors manually, you can create more complex test cases by using features such as the DataWeave expression language and context variables for saving values between calls to endpoints. 
```
Do we need to say anything about the alternative to creating monitors manually? IS there an alternative: default monitoring operations in Anypoint Monitoring? Have these already been explained? Should we begin this text with the advantage, something like "To enable customized and complex monitoring, you can create your own suite of tests, upload it, run it, and view the results..."?